x/gamm/types/account.go

```diff
@@ -52,16 +51,20 @@ func NewPoolAddress(poolId uint64) sdk.AccAddress {
 	return sdk.AccAddress(crypto.AddressHash(append(PoolAddressPrefix, sdk.Uint64ToBigEndian(poolId)...)))
 }
 
-func NewPoolAccount(poolId uint64, poolParams PoolParams, futureGovernor string) PoolAccountI {
+// NewPoolAccount returns a Balancer pool with the provided parameters, and initial assets.
+// Invariants that are assumed to be satisfied and not checked:
+// TODO: Why don't we check these in here?
+// TODO: Why does this panic, not just return an error?
```

Comment:
```suggestion
// TODO: Why does this panic, not just return an error?
```
Obsolete, I made it an error